my_game.py

```diff
@@ -400,22 +435,35 @@ def on_show_view(self):
         self.manager.enable()
 
         # Makes the play button.
-        self.gui_play_button = arcade.gui.UITextureButton(
-            x=150,
+        self.one_player = arcade.gui.UITextureButton(
+            x=50,
             y=125,
             width=100,
             height=100,
-            texture=arcade.load_texture("images/GUI/start_button_unhovered.png"),
-            texture_hovered=arcade.load_texture("images/GUI/start_button_hovered.png"),
-            scale=button_scaling,
+            texture=arcade.load_texture("images/GUI/one_player.png"),
```

Comment:
Could we rename the button files to something like _players_one_chosen.png_ & _players_one.png_.

Reply:
Do the same for other files used for buttons, so they all have the prefix _button__.